src/plugins/curator/components/gallery-fields/index.jsx

```diff
@@ -92,6 +92,7 @@ const GalleryFields = ({ isEditMode, gallery, formName, nodeRef, schemas }) => {
           name="allowComments"
         />
         <hr />
+        <Label style={{ display: 'block' }}>Related Galleries</Label>
```

Comment:
@cohencodes instead of doing an inline style, i encourage you to compare the working version and this version to see why they are different. you'll see that in the working version the picker component is wrapped in a FormGroup. so do that instead, make sure the other places where the galleryPickerField are used are not adversely affected, and do that instead